utils/apparmor/patches/120-gcc-13-compatibility.patch

```diff
@@ -0,0 +1,11 @@
+--- a/parser/capability.h
```

Comment:
please backport the upstream commit instead:
https://gitlab.com/apparmor/apparmor/-/commit/64a64be7ffb5a84f27daa9f37ae8ad92800943d3.patch
or bump this package to 3.0.10 or 3.1.4

Reply:
@1715173329

Done.